ql/test/experimental/frameworks/Gin/vendor/github.com/gin-gonic/gin/stub.go

```diff
@@ -259,7 +259,7 @@ func (_ *Context) IsAborted() bool {
 	return false
 }
 
-func (_ *Context) IsWebsocket() bool {
+func (_ *Context) IsWebSocket() bool {
```

Comment:
```suggestion
func (_ *Context) IsWebsocket() bool {
```

I assume this change was unintended?

Reply:
Whoops, yes; let me revert those changes.